gpbft/gpbft.go

```diff
@@ -567,26 +599,10 @@ func (i *instance) tryQuality() error {
 	return nil
 }
 
-func (i *instance) beginConverge() {
+func (i *instance) beginConvergeWithJustification(justification *Justification) {
```

Comment:
The logical change here seems fine, so I don't think we need to rename the method, just document the parameter.